src/main/java/io/aiven/kafka/auth/json/AivenKafkaPrincipalMapping.java

```diff
@@ -33,6 +44,8 @@ public static AivenKafkaPrincipalMapping forUnknownSslPrincipal() {
 
     /**
      * Checks if {@code sslPrincipal} matches against the regex of this entry.
+     * @param sslPrincipal - SSL principal
+     * @return boolean
```

Comment:
I would propose avoiding non-descriptive parameter comments, they only make it more difficult to read